config/locales/en/steps.yml

```diff
@@ -300,6 +300,15 @@ en:
       property_type:
         edit:
           page_title: Which assets does your client own or part-own inside or outside the UK?
+      savings:
+        edit:
+          page_title: Add savings account details 
+          heading: 
+            bank: Your client’s bank accounts
+            building_society: Your client’s building society accounts
+            cash_isa: Your client’s cash ISAs
+            national_savings_or_post_office: Your client’s National Savings or Post Office accounts
```

Comment:
Can we make all of these singular? I think it will help make it clearer to the user that they are adding one bank account at a time

Reply:
👍 still waiting on final content for these headings.

Reply:
Have updated as per your suggestion. Thank you :)